tests/eth_call_test.go

```diff
@@ -0,0 +1,60 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/stretchr/testify/require"
+)
+
+func TestEthCall_CodeLargerThanMaxInitCodeSizeIsNotAccepted(t *testing.T) {
+	tests := map[string]struct {
+		codeSize int
+		err      error
+	}{
+		"max code size": {
+			2 * 24576, // corresponds to the max init code size
+			nil,
+		},
+		"max code size + 1": {
+			2*24576 + 1,
+			fmt.Errorf("max code size exceeded"),
+		},
+	}
+	net := StartIntegrationTestNet(t)
+
+	client, err := net.GetClient()
+	if err != nil {
+		t.Fatalf("Failed to connect to the integration test network: %v", err)
+	}
+	defer client.Close()
+
+	rpcClient := client.Client()
+
+	for name, test := range tests {
+		t.Run(name, func(t *testing.T) {
+			hugeCode := make([]byte, test.codeSize)
+
+			params0 := map[string]string{
+				"to":   "0x5555555555555555555555555555555555555555",
+				"gas":  "0xffffffffffffffff",
+				"data": "0x00",
+			}
+			params1 := "latest"
+			params2 := map[string]map[string]hexutil.Bytes{
+				"0x5555555555555555555555555555555555555555": {
```

Comment:
```suggestion
                // The called account has code of the expected size.
				"0x5555555555555555555555555555555555555555": {
```
please add comment, otherwise is a little criptic

Reply:
I added a proper name to the address, I think this makes it more readable and has less chances of getting outdated.